content/600-about/200-prisma-docs/20-style-guide/07-user-interace-guidelines.mdx

```diff
@@ -0,0 +1,248 @@
+---
+title: 'User interface (GUI) guidelines'
+metaTitle: 'Prisma docs style guide: GUI guidelines'
+metaDescription: 'On this page you can find information about the Prisma guidelines for documenting UI elements and steps to walk users through interfaces.'
+tocDepth: 2
+---
+
+## Use numbered lists to document UI steps
+
+To document a complete task (such as, _Create a new project_), use numbered lists to document separate steps into the a sequence of steps. 
+
+* Use indented numbered lists to document substeps.
+* Avoid having tasks that comprise more than 10 steps or steps that comprise more than 10 substeps.
+* If you have more than 10 steps or steps with more than 5-10 substeps, consider separating into multiple tasks.
+
+## Write each step as a single action with a clear verb
```

Comment:
Could we have two more guidelines in this section along the following lines? Either here or elsewhere on this page.

**Guideline 1**
If you need to describe the results of a step, put them on a separate line beneath the step, indented at the same level as the step.

**Guideline 2**
Avoid describing obvious results that happen when the user carries out a step. For example, avoid this:

1. From the **Edit** menu, select **Copy**.
    The text is copied to the clipboard.